src/GameEngine.js

```diff
@@ -4,14 +4,19 @@ const Timer = require('./Timer');
 const EventEmitter = require('eventemitter3');
 
 class GameEngine{
-    constructor(inputOptions){
+    constructor(inputOptions, physicsEngine){
```

Comment:
ok, physicsEngine is now in the inputOptions.  Will be part of the upcoming pull request